backend/curriculum_tracking/helpers.py

```diff
@@ -182,3 +187,44 @@ def agile_card_reviews_outstanding(user):
 def pull_request_reviews_outstanding(user):
     # TODO: only implement this once the github webhook is healthier
     return []
+
+
+def get_formatted_duration_string(duration):
+    if duration:
+        duration = duration
+        seconds = duration.total_seconds()
+        days, remainder = divmod(seconds, 86400)
+        hours, remainder = divmod(remainder, 3600)
+        minutes, remainder = divmod(remainder, 60)
+
+        return f"{int(days)} days, {int(hours)} hours, {int(minutes)} minutes"
+
+    return None
+
+
+def get_blant_duration(id, card_status):
```

Comment:
The function should also probably just move to the agile card model, I struggle to see where it would actually be used other than when referring to the agilecards